app/presenters/idv/in_person/verify_info_presenter.rb

```diff
@@ -14,6 +14,6 @@ def identity_info_partial
   end
 
   def passport_flow?
-    @enrollment.passport_book?
+    @enrollment&.passport_book?
```

Comment:
It feels dangerous to allow a nil enrollment here since it in theory can't be. Can we look further to understand the cause better?